README.md

```diff
@@ -1,79 +1,26 @@
-# Base template for Gulp
+# Sibling tree
 
-Template to start from before starting frontend projects.
+## Task
 
-## Install template
+Create a tree that clicks on a headline hides-shows children:
 
-- Create empty repository for your project. [learn how](#) _need a link here_
-- Clone boilerplate into project directory and navigate there.
-```bash
-$ git clone https://github.com/mate-academy/gulp-template.git project-name
-$ cd project-name
-```
-- Change remote repository to recently created
-```bash
-$ git remote set-url origin github-link-of-project
-```
-- Install dependencies
-```bash
-$ npm install
-```
+Requirements: 
 
-All packages will be installed after executing the script above.
+- Use delegation.
+- Clicking outside the title text (in an empty place) should not do anything.
```

Comment:
double negation in english is not allowed. 

"should not do anything" -> "should do nothing".

Reply:
done